src/deepcode/view/SupportProvider.ts

```diff
@@ -0,0 +1,48 @@
+import { TreeItemCollapsibleState, ThemeIcon } from 'vscode';
+import { NodeProvider } from './NodeProvider';
+import { Node } from './Node';
+
+export class SupportProvider extends NodeProvider {
+  getRootChildren(): Node[] {
+    return [
+      new Node({
+        text: "Contact DeepCode",
+        description: "Send us a feedback or ask for support",
```

Comment:
Change this node to have only text: "Send us feedback or report a bug". Sounds like it would look like a bit simpler.